src/index.html

```diff
@@ -11,7 +11,178 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <div>
+          <label>
+            Surname:
+            <input
+              class="input"
+              type="text"
+              name="surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            Name:
+            <input
+              class="input"
+              type="text"
+              id="name"
+              name="name"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            How old are You?
+            <input type="number"
+              name="age"
+              min="1" max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="birthday"
+            >
+          </label>
+        </div>
+        <div class="input lastinput">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="agree"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <div class="input">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="input lastinput">
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+              minlength="6"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <div class="input">
+          <label>Do you love cats?</label>
+          <label>
+            <input
+              type="radio"
+              name="yes"
+            >
+            Yes
+          </label>
+          <label for="no">
```

Comment:
For is using with id
```suggestion
          <label>
```